Makefile

```diff
@@ -175,7 +175,12 @@ test-python: pyenv az
 	. pyenv/bin/activate && \
 		azdev linter && \
 		azdev style && \
-		hack/format-yaml/format-yaml.py .pipelines
+		hack/format-yaml/format-yaml.py .pipelines && \
+		cd python/az/aro && python -m unittest discover azext_aro/tests/latest/unit
+
+unit-test-python: pyenv az
+	. pyenv/bin/activate && \
```

Comment:
does this need to deactivate when done?

Reply:
Thanks @s-amann  for your review. 
Still a draft as I need to push some last commits to consider it "done".
Answering your question: it does not need to be deactivated as every Makefile recipe is executed in a [sub-shell](https://www.gnu.org/software/make/manual/html_node/Execution.html). This is why we don't need to worry when doing "cd" in the test-python or unit-test-python of the Makefile as it will not affect the current working directory.

Reply:
Thanks for the answer @AldoFusterTurpin, I didn't notice it was in draft until after I hit submit